<?xml version='1.0' encoding='UTF-8'?>
<!DOCTYPE dylanClass PUBLIC "-//Gwydion//DTD DITA Dylan API Class//EN" "../../../../dtd/dylanClass.dtd" []>
<dylanClass id="lib-network-sockets-Laddress_in_useG">
  <apiName>&lt;address-in-use&gt;</apiName>

  <shortdesc>The class of conditions signalled when a socket address
  is already in use.</shortdesc>

  <prolog>
    <author></author>
    <copyright>
      <copyryear year="2005"/>
      <copyrholder>Gwydion Dylan Maintainers</copyrholder>
    </copyright>
  </prolog>

  <dylanClassDetail>
    <dylanClassDef>
      <dylanSuperClass href="Lrecoverable_socket_conditionG.xml">&lt;recoverable-socket-condition&gt;</dylanSuperClass>
      <dylanSuperClass href="../sockets-internals/Lsealed_objectG.xml">&lt;sealed-object&gt;</dylanSuperClass>
      <dylanInitKeyword>
	<apiItemName>host-address:</apiItemName>
        <apiOtherClassifier href="../../dylan/dylan/LobjectG.xml">&lt;object&gt;</apiOtherClassifier>
	<apiDefNote></apiDefNote>
      </dylanInitKeyword>
      <dylanInitKeyword>
	<apiItemName>host-port:</apiItemName>
        <apiOtherClassifier href="../../dylan/dylan/LobjectG.xml">&lt;object&gt;</apiOtherClassifier>
	<apiDefNote></apiDefNote>
      </dylanInitKeyword>
    </dylanClassDef>

    <apiDesc>
      <p>The <apiname>&lt;address-in-use&gt;</apiname> condition is
      signalled when a process on the machine is already bound to the
      same fully qualified address. This condition probably occurred
      because you were trying to use a port with an active server
      already installed, or a process crashed without closing a
      socket.</p>
    </apiDesc>
  </dylanClassDetail>
</dylanClass>
